.github/workflows/ci.yml

```
@@ -115,6 +115,14 @@ jobs:
         with: { bun-version: "${{ env.BUN_VERSION }}" }
       - run: bun install --frozen-lockfile
       - run: bun run test:cli
+      - if: runner.os == 'Linux'
+        run: sudo apt-get update
+      - if: runner.os == 'Linux'
+        run: sudo apt-get install --no-install-recommends -y dbus-x11 gnome-keyring libsecret-tools
+      - if: runner.os == 'Linux'
+        run: dbus-run-session -- bash -c 'eval "$(gnome-keyring-daemon --start --components=secrets)" && export GNOME_KEYRING_CONTROL SSH_AUTH_SOCK && bun run test:cli:live'
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2016:info:1:29: Expressions don't expand in single quotes, use double quotes for that

(shellcheck)